event/src/unvalidated/payload/signed.rs

```diff
@@ -6,6 +6,7 @@ use std::collections::BTreeMap;
 
 /// A signed event payload.
 #[derive(Debug, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
```

Comment:
this is the only one I'm not sure of.  I believe this struct represents the JWS envelope and I don't actually know if it's safe to assume that that doesn't have extra fields.  I think I'm leaning towards removing the deny_unknown_fields tag on this struct since the JWS envelope format isn't really a part of the Ceramic protocol spec so feels like maybe a place where we should be looser in what we accept.  Curious for other's thoughts here.

Reply:
Wait, actually I'm confused.  There's also the `SignedEvent` struct that has a `Jws` struct: https://github.com/ceramicnetwork/rust-ceramic/blob/c803ac49ec7b136a3aca42927c5696649238841a/event/src/unvalidated/ext/signed_event.rs#L17

@dbcfd - what's the relationship between this signed::Payload struct and the SignedEvent struct?  Are they both used or can one be deleted?

Reply:
SignedEvent is what js-ceramic requires, hence it is in ext. signed::Payload is what comes from js-ceramic when it is written to a car file.